src/video/sdl2_v.cpp

```diff
@@ -243,6 +243,19 @@ void VideoDriver_SDL_Base::EditBoxLostFocus()
 	}
 }
 
+std::vector<int> VideoDriver_SDL_Base::GetListOfMonitorRefreshRates() {
+	std::vector<int> rates = {};
+	int displayCount = SDL_GetNumVideoDisplays();
+	for (int i = 0; i < displayCount; i++) {
+		SDL_DisplayMode mode = {};
+		if (SDL_GetDisplayMode(i, 0, &mode) != 0) {
+			// error?
```

Comment:
no; just ignore it. So `continue`. As currently on error it also adds the (last?) entry to the list :D